bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -280,6 +281,7 @@ private Addon convertTopicItemToAddon(DiscourseTopicItem topic, List<DiscourseUs
         String id = ADDON_ID_PREFIX + topic.id.toString();
         AddonType addonType = types.get(topic.category_id);
         String type = (addonType != null) ? addonType.getId() : "";
+        // TODO: content-type differentiation for kar and jar
```

Comment:
Is there still something to be done here?

Reply:
Yes, that's the one open question. Currently we distinguish between kar and jar by looking at the extension of the filename from the download link. Unfortunately that is not available here, , because it is not present in the `DiscourseTopicItem`. This method is used in `getAddons` which lists all addons. In `getAddon`, which only gets the addon for a single addon id, we have a `DiscourseTopicResponseDTO` which includes that field.

Reply:
So this is still TBD, see also https://github.com/openhab/openhab-core/pull/2405#issuecomment-922330139

Reply:
I have renamed the Bindings category to Bundles and added the following tag groups:

![image](https://user-images.githubusercontent.com/2004147/133897001-779aba51-f380-4670-8927-a915aa4c9740.png)

⬆️ it would be nice to have this added as a "maturity" property if one of these are found. This could come in handy later.

![image](https://user-images.githubusercontent.com/2004147/133897009-6fed2784-560f-4bdf-90ab-1e30f058cda9.png)

Also the category is configured like this:

![image](https://user-images.githubusercontent.com/2004147/133897110-4aba674a-53f0-4f49-a622-61b653595d73.png)

Reply:
Thanks @ghys, that should work and give us everything we want for the forseeable future. :slightly_smiling_face::crossed_fingers:

Reply:
I'll implement that